src/persist-client/src/internal/state.proto

```diff
@@ -202,11 +202,15 @@ message ProtoRollup {
     map<string, ProtoLeasedReaderState> leased_readers = 8;
     map<string, ProtoCriticalReaderState> critical_readers = 13;
     map<string, ProtoWriterState> writers = 9;
-    map<uint64, ProtoEncodedSchemas> schemas = 18;
+    map<uint64, ProtoEncodedSchemas> schemas = 19;
 
     ProtoInlinedDiffs diffs = 17;
 
     // MIGRATION: We previously stored rollups as a `SeqNo -> string Key` map,
     // but now the value is a `struct HollowRollup`.
     map<uint64, string> deprecated_rollups = 12;
+    // MIGRATION: We previously registered schemas that have since changed
+    // nullability. We've made changes to stabilize the schemas and are just
+    // forgetting all of the old values.
+    reserved 18;
```

Comment:
Hmm, I think we have to do this also for StateDiff. And then once that's done, I worry about getting "state diff does not apply cleanly errors" if we just forget the old fields. It's counter-intuitive I know, but I do think this is less risky if we rename all the old fields, but keep all the diff apply logic for them

Reply:
Yeah, probably safest to add them both back. Looks like they still exist in LD. I'd probably just pull the old definitions out of git history and then not even bother rolling them out slowly, just default to on